Makefile

```diff
@@ -7,5 +7,46 @@ integration:
 	behave --tags=$(INTEGRATIONS) tests -f progress2
 
 PYTHON_VERSION ?= 3.8
+docker-test: export PYTHON_VERSION := $(PYTHON_VERSION)
+docker-test: export DOCKER_TESTING ?= true
+docker-test: export TESTING_URL ?= https://github.com/algorand/algorand-sdk-testing.git
+docker-test: export TESTING_BRANCH ?= control-my-test-branches # revert to master before merge
 docker-test:
-	PYTHON_VERSION='$(PYTHON_VERSION)' ./run_integration.sh
+	./integration.sh
+
+docker-ps:
+	cd test-harness && docker-compose ps
+
+docker-pause:
+	cd test-harness && docker-compose pause
+
+docker-unpause:
+	cd test-harness && docker-compose unpause
+
+# these are utilized if run locally:
+LOCAL_TESTENV ?= .local-env
+prep-local-testenv:
+	python tests/prep_local_testenv.py $(LOCAL_TESTENV)
+
+show-local-testenv:
+	echo "contents of $(LOCAL_TESTENV)"
+	cat $(LOCAL_TESTENV) || echo "DNE"
+
+# configure the repos and branches that a local test will build off of:
+build-for-local-test: export TESTING_URL ?= https://github.com/algorand/algorand-sdk-testing.git
+build-for-local-test: export TESTING_BRANCH ?= control-my-test-branches # revert to master before merge
+build-for-local-test: export ALGOD_URL ?= https://github.com/algorand/go-algorand
+build-for-local-test: export ALGOD_BRANCH ?= feature/avm-box # revert to master before merge
+build-for-local-test: export INDEXER_URL ?= https://github.com/algorand/indexer
+build-for-local-test: export INDEXER_BRANCH ?= localledger/integration # revert to develop before merge
+
+# other exports for `local-test`:
+build-for-local-test: export PYTHON_VERSION := $(PYTHON_VERSION)
+build-for-local-test: export LOCAL_TESTENV := $(abspath .)/$(LOCAL_TESTENV)
+build-for-local-test: export DOCKER_TESTING ?= false
+build-for-local-test: export TYPE_OVERRIDE ?= source
+
+# local-test bootstraps the necessary algod and indexer docker containers
+# for integration tests, but does not run the tests
```

Comment:
Consider amending this with "unless `DOCKER_TESTING` is true" to reduce potential confusion